packages/legend-graph/src/graphManager/protocol/pure/v1/model/packageableElements/store/flatData/model/V1_FlatDataProperty.ts

```diff
@@ -19,13 +19,13 @@ import { type Hashable, hashArray } from '@finos/legend-shared';
 
 export class V1_FlatDataProperty implements Hashable {
   name!: string;
-  value!: boolean | string | number;
+  value!: (boolean | string | number)[];
 
   get hashCode(): string {
     return hashArray([
       CORE_HASH_STRUCTURE.FLAT_DATA_PROPERTY,
       this.name,
-      this.value.toString(),
+      this.value.map((v) => v.toString()).join(','),
```

Comment:
Let's just update to use `hashArray` for these places

Reply:
this.value has type (boolean | string | number)[] but hashArray is defined on (string | Hashable)[] 

Do you want me to change hashArray?

Reply:
Yeah and you can call toString on them, I prefer consistency and I think joining strings by , introduce edge cases

Reply:
done as suggested